bitsail-connectors/connector-doris/src/main/java/com/bytedance/bitsail/connector/doris/option/DorisWriterOptions.java

```diff
@@ -78,11 +79,11 @@ public interface DorisWriterOptions extends WriterOptions.BaseWriterOptions {
 
   ConfigOption<Integer> SINK_BUFFER_SIZE =
       key(WRITER_PREFIX + "sink_buffer_size")
-          .defaultValue(20 * 1024 * 1024);
+          .defaultValue(1024 * 1024);
 
   ConfigOption<Integer> SINK_BUFFER_COUNT =
       key(WRITER_PREFIX + "sink_buffer_count")
-          .defaultValue(100000);
+          .defaultValue(3);
```

Comment:
Could you use new options? These two options are used in traditional replace proxy, I don't think the default value can be changed.

Reply:
done, i use `SINK_UPSERT_BUFFER_SIZE` and `SINK_UPSERT_BUFFER_COUNT` to repalce